specification/dartLangSpec.tex

```diff
@@ -15930,39 +15938,6 @@ \subsection{Lexical Lookup}
 That situation may cause an error, as specified below.%
 }
 
-\LMHash{}%
-\Case{$D$ does not exist}
-When no declaration with basename \id{} is in scope at the location $\ell$,
-the following errors apply:
-
-\begin{itemize}
-\item
-  % Here is the list of cases covered by this item: \ell is..
-  % - inside a library function, getter, setter, variable initializer;
-  % - inside a static method, getter, setter, variable initializer;
-  % - inside an instance variable initializer;
-  % - in an expression in the initializer list of a constructor
-  %   (be it a superinitializer, a field initializer, or an assertion).
-  It is a compile-time error if $\ell$ does not have access to \THIS{}
-  (\ref{classes}).
-\item
-  When $\ell$ has access to \THIS{}
-  it is a compile-time error if $\ell$ occurs in
-  a class \DefineSymbol{C} and
-  the interface of $C$ does not have a member named $n$;
-  and it is a compile-time error if $\ell$ occurs in
-  an extension \DefineSymbol{E},
-  and the interface of the \ON{} type of $E$ does not have a member named $n$.
-  \commentary{%
-    So it is an error if there is no member with the right basename at all,
-    and also if there is one such member,
-    but it is a non-setter and we are looking for a setter,
-    or vice versa.%
-  }
-\end{itemize}
-\vspace{-2ex}
-\EndCase
-
 % Even when we have found a declaration, it may have the wrong name.
 \LMHash{}%
 \Case{$D$ exists}
```

Comment:
As below, what is D?

Reply:
Never mind - this seems to be something broken with github's preview.  It's showing line 15876 followed immediately by line 15938 with no option to expand the lines between, and I think that's where the missing text is.  I think I'm going to have to pull this down to my machine to review tomorrow.

Reply:
Yes, I've been looking for better expansion support in github reviews, too.